react/features/base/ui/components/GlobalStyles.js

```diff
@@ -15,6 +15,12 @@ import { formatCommonClasses } from '../functions';
 const useStyles = makeStyles(theme =>
     createStyles({
         '@global': {
+            ':root': {
+                /**
+                * Prejoin / premeeting screen.
+                */
+                '--prejoin-default-content-width': 336
```

Comment:
Please move this to a JS const and a comment `Maps SCSS variable $prejoinDefaultContentWidth`

Reply:
@robertpin  Should I remove the css variable and use a javascript const or store the value in a JS const and use it in css?

```
const PREJOIN_DEFAULT_CONTENT_WIDTH = 336;
```
```
'--prejoin-default-content-width': PREJOIN_DEFAULT_CONTENT_WIDTH
```

Reply:
Just use the JS const in JSS and remove the `--prejoin-default-content-width`

Reply:
It's not ideal, but for now we will have two variables for the same thing, one in JS and one in SASS. The idea is that we hope to eventually move more code to JSS and use the JS const

Reply:
And because we can use JS const in JSS there's no need to create a CSS var to hold the JS const

Reply:
Storing in a JSS const would have the overhead of imports. I think using CSS variables are more convenient than JS consts, but it is your call.

Reply:
You have a good point, but for now let's go with import, as this is how it's done for other components. I prefer we keep the global styles clean for now, but we'll consider moving them in the future. Thank you